src/Weather/Rasp/RaspStyle.cpp

```diff
@@ -17,7 +17,7 @@ Copyright_License {
 
   You should have received a copy of the GNU General Public License
   along with this program; if not, write to the Free Software
-  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1306, USA.
```

Comment:
.... ???

Reply:
I'm just as puzzled. That shouldn't be there.